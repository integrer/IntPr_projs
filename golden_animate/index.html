
<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Golden</title>

	<link rel="stylesheet" href="css/style.css">
	<link rel="stylesheet" href="css/responsive.css">
	<link rel="stylesheet" href="css/font-awesome.css">
	<link rel="stylesheet" href="css/animate.min.css">

	<script src="js/wow.min.js"></script>
	<script>new WOW().init();</script>

</head>
<body>
	<header>
		<div class="container">
			<div class="heading clearfix">
				<img src="img/logo.png" alt="Golden" class="logo">
				<nav>
					<ul class="menu">
						<li><a href="#">Home</a></li>
						<li><a href="#services">Services</a></li>
						<li><a href="#portfolio">Portfolio</a>	</li>
						<li><a href="#">About</a></li>
						<li><a href="#">Contact</a></li>
					</ul>
				</nav>
			</div>
			<div class="titles">
				<div class="titles__first wow fadeInLeft" data-wow-duration="0.75s">Welcome To Our Studio!</div>
				<h1 class="wow fadeInRight" data-wow-duration="0.75s"> It’s nice to Meet you</h1>
			</div>
			<a class="button" href="#">Tell Me More</a>
		</div>
	</header>
	<section id="services">
		<div class="container">
			<div class="title">
				<h2>Services</h2>
				<p>Proin iaculis purus consequat sem cure.</p>
			</div>
			<div class="services clearfix">
				<div class="container__30item wow	zoomInDown">
					<img src="img/icon1.png" alt="Услуга">
					<h3>E-Commerce</h3>
					<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit
						aenean rhoncus posuere odio in tincidunt.</p>
				</div>
				<div class="container__30item wow	zoomInDown">
					<img src="img/icon2.png" alt="Услуга">
					<h3>Responsive Web</h3>
					<p>
						Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit
						aenean rhoncus posuere odio in tincidunt.
					</p>
				</div>
				<div class="container__30item wow	zoomInDown">
					<img src="img/icon3.png" alt="Услуга">
					<h3>Web Security</h3>
					<p>
						Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit
						aenean rhoncus posuere odio in tincidunt.
					</p>
				</div>
			</div>
		</div>
	</section>
	<section id="portfolio">
		<div class="container">
			<div class="title">
				<h2>Our portfolio</h2>
				<p>Proin iaculis purus consequat sem cure.</p>
			</div>
			<div class="works clearfix">
				<div class="container__30item">
					<img src="img/img1.jpg" alt="Работа">
					<h3>Ebony & Verony</h3>
					<h4>Branding</h4>
				</div>
				<div class="container__30item">
					<img src="img/img2.jpg" alt="Работа">
					<h3>Smart Stationary</h3>
					<h4>Print Design</h4>
				</div>
				<div class="container__30item">
					<img src="img/img3.jpg" alt="Работа">
					<h3>Clever Post</h3>
					<h4>Print Design</h4>
				</div>
			</div>
			<div class="works clearfix">
				<div class="container__30item">
					<img src="img/img1.jpg" alt="Работа">
					<h3>Ebony & Verony</h3>
					<h4>Branding</h4>
				</div>
				<div class="container__30item">
					<img src="img/img2.jpg" alt="Работа">
					<h3>Smart Stationary</h3>
					<h4>Print Design</h4>
				</div>
				<div class="container__30item">
					<img src="img/img3.jpg" alt="Работа">
					<h3>Clever Post</h3>
					<h4>Print Design</h4>
				</div>
			</div>
		</div>
	</section>
	<section id="about">
		<div class="container">
			<div class="title">
				<h2>About us</h2>
				<p>Proin iaculis purus consequat sem cure.</p>
			</div>
			<div class="about clearfix">
				<div class="container__50itemleft">
          <div><img src="img/about1.jpg" alt="Работа"></div>
				  <h3>July 2010<br>Our Humble Beginnings</h3>
					<p>Proin iaculis purus consequat sem cure
					digni ssim. Donec porttitora entum suscipit
					aenean rhoncus posuere odio in tincidunt. Proin
					iaculis purus consequat sem cure digni
					ssim. </p>


				</div>
        <div class="clearfix"></div>
				<div class="container__50itemright">
					<div><img src="img/about2.jpg" alt="Работа"></div>
					<h3>January 2011<br>Facing Startup Battles</h3>
					<p>Proin iaculis purus consequat sem cure
					digni ssim. Donec porttitora entum suscipit
					aenean rhoncus posuere odio in tincidunt. Proin
					iaculis purus consequat sem cure digni
					ssim. </p>
				</div>

				<div class="container__50itemleft">

					 <div><img src="img/about3.jpg" alt="Работа"></div>
					<h3>December 2012 <br>Enter The Dark Days</h3>
					<p>Proin iaculis purus consequat sem cure
					digni ssim. Donec porttitora entum suscipit
					aenean rhoncus posuere odio in tincidunt. Proin
					iaculis purus consequat sem cure digni
					ssim. </p>

			  </div>

				<div class="container__50itemright">
					<div><img src="img/about4.jpg" alt="Работа"></div>
					<h3>February 2014<br>Our Triumph</h3>
					<p>Proin iaculis purus consequat sem cure
					digni ssim. Donec porttitora entum suscipit
					aenean rhoncus posuere odio in tincidunt. Proin
					iaculis purus consequat sem cure digni
					ssim.
				  </p>
				</div>
				<div class="clearfix"></div>
        <a class="circle_button shadow" href="#">OUR<br>STORY<br>CONTINUES<br>...</a>
			</div>
  </section>
  <section id="team">
		<div class="container">
			<div class="title">
				<h2>Our amazing team</h2>
				<p>Proin iaculis purus consequat sem cure.</p>
			</div>
			<div class="works clearfix">
				<div class="container__30item">
					<img src="img/pic.jpg" alt="" class="round shadow scale-img">
					<h3>Kimberly Thompson</h3>
					<h4>Marketer</h4>
					<i class="fa fa-twitter fa-radius fa-2x"></i> <i class="fa fa-facebook fa-radius fa-2x"></i> <i class="fa fa-pinterest-p fa-radius fa-2x"></i>
				</div>
				<div class="container__30item">
					<img src="img/pic2.jpg" alt="" class="round shadow">
					<h3>Rico Massimo</h3>
					<h4>Coder</h4>
					<i class="fa fa-odnoklassniki fa-radius fa-2x"></i> <i class="fa fa-facebook fa-radius fa-2x"></i> <i class="fa fa-pinterest-p fa-radius fa-2x"></i>
				</div>
				<div class="container__30item">
					<img src="img/pic4.jpg" alt="" class="round shadow">
					<h3>Uku Masson</h3>
					<h4>Graphic Designer</h4>
					  <i class="fa fa-twitter fa-radius fa-2x"></i> <i class="fa fa-facebook fa-radius fa-2x"></i> <i class="fa fa-pinterest-p fa-radius fa-2x"></i>
				</div>
			</div>
			<div class="text_team">
				Proin iaculis purus consequat sem cure
				digni ssim. Donec porttitora entum suscipit
				aenean rhoncus posuere odio in tincidunt. Proin
				iaculis purus consequat sem cure digni
				ssim.
			</div>
		</div>
	</section>
	<section class="brand">
		<div class="container clearfix">
			<div class="brand">
				<div class="container__25item"><img src="img\Envato.jpg" alt=""></div>
				<div class="container__25item"><img src="img\WordPress.jpg" alt=""></div>
				<div class="container__25item"><img src="img\Tuts+.jpg" alt=""></div>
				<div class="container__25item"><img src="img\Microlancer.jpg" alt=""></div>
			</div>
		</div>
	</section>
	<section id="contact">
		<div class="container">
			<div class="title">
				<h2 class="white">Contact Us</h2>
				<p>Proin iaculis purus consequat sem cure.</p>
			</div>
			<div class="contact clearfix">
				<form class="formit" name="test" method="post" action="input1.php">
					<div class="container_50item">
						<input name="name" placeholder="YOUR NAME*" type="text">
						<input name="email" placeholder="YOUR EMAIL*" type="text">
						<input name="subject" placeholder="SUBJECT" type="text">
					</div>
					<div class="container_50item">
						<textarea name="comment" placeholder="YOUR MESSAGE"></textarea>
					</div>
				</form>
			</div>
			<a class="button" href="#">send MESSAGE</a>
		</div>
	</section>

	<footer>
		<div class="container">
			<div class="footer">
				<div class="copyright">	© Copyright 2017. Все права защищены.</div>
					<a href="#"><span class="fa-stack fa-lg"><i class="fa fa-circle fa-stack-2x"></i><i class="fa fa-facebook-f fa-stack-1x fa-inverse"></i></span></a>
					<a href="#"><span class="fa-stack fa-lg"><i class="fa fa-circle fa-stack-2x"></i><i class="fa fa-twitter fa-stack-1x fa-inverse"></i></span></a>
					<a href="#"><span class="fa-stack fa-lg"><i class="fa fa-circle fa-stack-2x"></i><i class="fa fa-odnoklassniki fa-stack-1x fa-inverse"></i></span></a>
					<a href="#"><span class="fa-stack fa-lg"><i class="fa fa-circle fa-stack-2x"></i><i class="fa fa-pinterest-p fa-stack-1x fa-inverse"></i></span></a>
			</div>
		</div>
	</footer>


</body>
</html>
