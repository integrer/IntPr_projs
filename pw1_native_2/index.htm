<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
  <link rel="stylesheet" href="css/main.css">
  <title>Document</title>
</head>
<body>
  <div class="grad_bg">
  <header>
    <br>
    <br>
    <br>
    <br>
    <br>
    <br>
    <br>
    <br>
    <br>
    <br>
  </header>
  </div>
  <section class="yellow_bg">
    <br>
    <br>
    <br>
    <br>
    <br>
    <br>
    <br>
    <br>
    <br>
    <br>
  </section>
  <section></section>
  <section></section>
  <section></section>
  <footer></footer>
</body>
</html>
