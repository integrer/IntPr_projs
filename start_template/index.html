<!DOCTYPE html>
<html lang="ru">
<head>
	<meta charset="UTF-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge" />
	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
	<!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->

	<!-- Titles -->
	<title>Document</title>
	<meta name="keywords" content="">
    <meta name="description" content="">

    <!-- Bootstrap CSS -->
	<link rel="stylesheet" href="libs/bootstrap/css/bootstrap.min.css">
	<!-- Styles -->
	<link rel="stylesheet" href="css/style.css">

	<!-- FavIcon -->
	<link rel="icon" href="img/favicon.png"  type="image/x-icon" />



</head>
<body>

		<!-- Здесь пишем код -->

	<section>
		<div class="container">
			<h1 class = "display-3 text-center">Заказывайте визитки у нас!!!</h1>
			<div class="row">
				<div class="col-5">
					<div class="calc">
						<h2 class = "display-5">Расчет стоимости</h2><br>
						<h4 class = "display-6">Тираж</h4>
						<select class="form-control" name="edition" id="card-edition">
							<option value="90">90</option>
							<option value="120">120</option>
							<option value="150">150</option>
							<option value="180">180</option>
							<option value="210">210</option>
							<option value="240">240</option>
							<option value="270">270</option>
							<option value="300">300</option>
							<option value="360">360</option>
							<option value="420">420</option>
							<option value="510">510</option>
							<option value="600">900</option>
							<option value="1020">1020</option>
							<option value="1500">1500</option>
							<option value="3000">3000</option>
							<option value="6000">6000</option>
						</select><br>

						<h4 class = "display-6">Бумага</h4>
						<select class="form-control" name="card_paper" id="card_paper">
							<option value="Мелованная бумага (белая)" data-paper-pice="6.8">Мелованная бумага (белая)</option>
							<option value="Бумага повышеной белезны" data-paper-pice="40.5">Бумага повышеной белезны</option>
							<option value="Prestige Лён (белый)" data-paper-pice="45">Prestige Лён (белый)</option>
						</select><br>

						<h4 class = "display-6">Цветность</h4>
						<select class="form-control" name="card_color" id="card_color">
							<option value="Односторонняя черно-белая" data-color="0" data-color-file="110" data-color-imprint="14">Односторонняя черно-белая</option>
							<option value="Двусторонняя черно-белая" data-color="1" data-color-file="220" data-color-imprint="28">Двусторонняя черно-белая</option>
							<option value="Односторонняя цветная" data-color="2" data-color-file="110" data-color-imprint="37">Односторонняя цветная</option>
							<option value="Цветная с лицевой, ч\б с оборотной" data-color="3" data-color-file="220" data-color-imprint="51">Цветная с лицевой, ч\б с оборотной</option>
							<option value="Двусторонняя цветная" data-color="4" data-color-file="220" data-color-imprint="74">Двусторонняя цветная</option>
						</select><br>

						<h4 class = "text-danger display-5"><small>Приблизительная</small> <br>Стоимость за тираж</h4>
						<p class = "price">
							<span id = "final_price">0</span> руб.
						</p>
					</div>
				</div>
				<div class="col-7">
					<div class="card-images text-center">
						<img src="img/1_0.png" alt="" id="card_1_0">
						<img src="img/1_1.png" alt="" id="card_1_1">
						<img src="img/4_0.png" alt="" id="card_4_0">
						<img src="img/4_1.png" alt="" id="card_4_1">
						<img src="img/4_4.png" alt="" id="card_4_4">
					</div>
				</div>
			</div>
		</div>
	</section>




	<!-- Библиотека jQuery - необходима для Bootstrap плагинов -->
	<script src="libs/jquery/jquery.min.js"></script>

	<!-- Bootstrap JS скрипты -->
	<script src="libs/bootstrap/js/bootstrap.min.js"></script>

	<!-- Собственные JS скрипты -->
	<script src="js/main.js"></script>

</body>
</html>
