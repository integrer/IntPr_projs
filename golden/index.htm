<!DOCTYPE html>

<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<meta http-equiv="X-UA-Compatible" content="ie=edge">

	<!-- CSS -->
	<link rel="stylesheet" href="css/main.css">
	<link rel="stylesheet" href="css/responcive.css">
	<link rel="stylesheet" href="css/font-awesome.min.css">

	<title>Document</title>
</head>

<body>
	<header>
		<div class="container">
			<div class="clearfix">
				<img class="logo" src="img/logo.png" alt="Golden"/>
				<nav>
					<ul class="menu">
						<li><a href="#">Home</a></li>
						<li><a href="#">Services</a></li>
						<li><a href="#">Portfolio</a></li>
						<li><a href="#">About</a></li>
						<li><a href="#">Contact</a></li>
					</ul>
				</nav>
			</div>
			<div class="titles">
				<h3>Welcome To Our Studio!</h3>
				<h1>It`s nice to meet you</h1>
				<a href="#" class="button">Tell me more</a>
			</div>
		</div>
	</header>

	<section class="#">				<!--1 Services-->
		<div class="container">
			<div class="title">
				<h2>Services</h2>
				<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Laboriosam ea tempora minus! Laboriosam aperiam, cumque! Rerum mollitia ullam nemo facilis molestiae. Aliquam ratione sequi ipsum nobis impedit, odio quos repellat.</p>
			</div>

			<div class="clearfix">
				<div class="item_rect">
					<img src="img/icon1.png" alt="E-Commerce">
					<h4>E-Commerce</h4>
					<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
				</div>

				<div class="item_rect">
					<img src="IMG/icon2.png" alt="Responsive Web">
					<h4>Responsive Web</h4>
					<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
				</div>

				<div class="item_rect">
					<img src="IMG/icon3.png" alt="Web Security">
					<h4>Web Security</h4>
					<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
				</div>
			</div>
		</div>
	</section>

	<section class="gray_bg">	<!--2 Our portfolio-->
		<div class="container">
			<div class="title">
				<h2>Our portfolio</h2>
				<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
			</div>

			<div class="clearfix">
				<div class="item_rect">
					<img src="img/img1.jpg" alt="">
					<h4>Ebony & Ivory</h4>
					<p>Branding</p>
				</div>
				<div class="item_rect">
					<img src="img/img2.jpg" alt="">
					<h4>Smart Statigionary</h4>
					<p>Print Design</p>
				</div>
				<div class="item_rect">
					<img src="img/img3.jpg" alt="">
					<h4>clever Poster</h4>
					<p>Print Design</p>
				</div>
			</div>

			<div class="clearfix">
				<div class="item_rect">
					<img src="img/img4.jpg" alt="">
					<h4>Vinyl Record</h4>
					<p>Product Mock-Up</p>
				</div>
				<div class="item_rect">
					<img src="img/img5.jpg" alt="">
					<h4>Treehouse Template</h4>
					<p>Web Design</p>
				</div>
				<div class="item_rect">
					<img src="img/img6.jpg" alt="">
					<h4>clever Poster</h4>
					<p>Branding</p>
				</div>
			</div>
		</div>
	</section>

	<section class="#">				<!--3 About us-->
		<div class="container">
			<div class="title">
				<h2>About us</h2>
				<p></p>
			</div>
			<div class="clearfix">
				<div class="item_radial">
					<div><img src="img/about1.jpg" alt=""></div>
					<h4>JULY 2010<br>Our Humble Beginnings</h4>
					<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. In ipsa mollitia rem eum, dignissimos accusamus ab harum maxime magni repudiandae. Officia, quis id assumenda, similique commodi vitae odit totam praesentium.</p>
				</div>
				<div class="item_radial lft">
					<div><img src="img/about2.jpg" alt=""></div>
					<h4>JANUARY 2011<br>Facing Startup Buttles</h4>
					<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Incidunt dolore architecto, possimus asperiores minus ut voluptatibus nulla non impedit ab voluptatem est quae repellat assumenda quisquam, consequuntur esse soluta omnis!</p>
				</div>
				<div class="item_radial">
					<div><img src="img/about3.jpg" alt=""></div>
					<h4>DECEMBER 2012<br>Enter The Dark Days</h4>
					<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aspernatur sequi et impedit ducimus harum delectus voluptatem alias iusto. Culpa quas id vel corporis nihil repellat, numquam pariatur dolor dolore. Repudiandae.</p>
				</div>
				<div class="item_radial lft">
					<div><img src="img/about4.jpg" alt=""></div>
					<h4>FEBRUARY 2104<br>Our Triumph</h4>
					<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ipsam ducimus dicta ea ipsa consequuntur deserunt fugiat laudantium sit quia quisquam non quo, placeat expedita doloribus eaque qui quas. Necessitatibus, perferendis?</p>
				</div>
			</div>
			<a href="#" class="button radial_btn">OUR<br>STORY<br>CONTINUES<br>...</a>
		</div>
	</section>

	<section class="gray_bg">	<!--4 Our amazing team-->
		<div class="container">
			<div class="title">
				<h2>Our amazing team</h2>
				<p></p>
				<div class="clearfix">
					<div class="item_radial_line">
						<img src="img/pic4.jpg" alt="E-Commerce">
						<h4>E-Commerce</h4>
						<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
					</div>

					<div class="item_radial_line">
						<img src="IMG/pic2.jpg" alt="Responsive Web">
						<h4>Responsive Web</h4>
						<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
					</div>

					<div class="item_radial_line">
						<img src="IMG/pic.jpg" alt="Web Security">
						<h4>Web Security</h4>
						<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
					</div>

			</div>
		</div>
	</section>

	<section class="#">				<!--5 envato-->
		<div class="container">
		</div>
	</section>

	<section class="#">				<!--6 Contact us-->
		<div class="container">
			<div class="title">
				<h2>Contact us</h2>
				<p></p>
			</div>
		</div>
	</section>

	<footer></footer>
</body>
</html>
